src/tmx/TmxUtils/CMakeLists.txt

```diff
@@ -1,13 +1,11 @@
-PROJECT ( tmxutils CXX )
-FILE (GLOB_RECURSE HEADERS "src/" "*.h*")
-FILE (GLOB_RECURSE SOURCES  "src/" "*.c*")
+project( tmxutils CXX )
+file(GLOB_RECURSE HEADERS "src/" "src/*.h*")
```

Comment:
Do we still need the first "src"? It seems to be ignored.

Reply:
"src/*.h*" shall already specify which folder to look at. Where in the documentation indicates that  additional "src" before wildcard is required?

Reply:
<img width="494" height="41" alt="image" src="https://github.com/user-attachments/assets/fc103005-779d-48d9-b166-f44230c4fe21" />
Yeah, it seems "src/*.h*" shall be sufficient per the above example `/dir/*py`